src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,68 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.util.Scanner;
+
 public class WorkWithFile {
+    private static final String FIRST_OPERATION_NAME = "supply";
+    private static final String SECOND_OPERATION_NAME = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] textLines = getInputText(fromFileName).split(System.lineSeparator());
+        int[] amountCounter = new int[2]; //supply - index 0, buy - index 1
+        for (String line : textLines) {
+            String[] lineInfo = line.split(",");
+            if (lineInfo[0].equals(FIRST_OPERATION_NAME)) {
+                amountCounter[0] += Integer.parseInt(lineInfo[1]);
+            } else {
+                amountCounter[1] += Integer.parseInt(lineInfo[1]);
+            }
+        }
+        String inputText = getReportText(amountCounter);
+        writeToFile(toFileName, inputText);
+    }
+
+    private static void writeToFile(String fileName, String inputText) {
+        File file = new File(fileName);
+        try {
+            file.createNewFile();
+        } catch (java.io.IOException e) {
```

Comment:
let's import IOException class
```suggestion
        } catch (IOException e) {
```

Reply:
done